models.py

```diff
@@ -946,7 +946,7 @@ class FeatureForm(forms.Form):
   name = forms.CharField(required=True, label='Feature',
       help_text='Capitalize only the first letter and the beginnings of proper nouns.')
 
-  summary = forms.CharField(label='', required=True, max_length=500,
```

Comment:
Do you know exactly what the `max_length` field did? This seems redundant, but is it?

Reply:
All of the reading I did suggested they're redundant, so AFAIK there's no material difference between the two. I went with `maxlength` in the textarea attrs for consistency with the surrounding code, but I don't have a strong feeling either way here.